src/components/interactive-image/interactive-image.tsx

```diff
@@ -0,0 +1,222 @@
+import { Component, Element, Prop, h, State, Watch } from "@stencil/core";
+import { Component as GxComponent } from "../common/interfaces";
+
+@Component({
+  shadow: false,
+  styleUrl: "interactive-image.scss",
+  tag: "gx-interactive-image"
+})
+export class InteractiveImage implements GxComponent {
+  @Element() element: HTMLGxInteractiveImageElement;
+
+  /**
+   * If this property is true, the user can copy the image to the clipboard (Only for iOS, see image below).
+   */
+  @Prop() enableCopyToClipboard: false;
+
+  /**
+   * True/False. If this property is true, the user can zoom in/out on the image.
+   */
+  @Prop() enableZoom = false;
+
+  /**
+   * Indicates how much you can enlarge an image. (Percentage) _Note: 100% = Normal size_.
+   */
+  @Prop() zoom = 100;
+
+  /**
+   * Indicates how much you can enlarge an image with reference to the original size of it or the size of the controller. (Percentage)
+   */
+  @Prop() maxZoomRelativeTo: number;
```

Comment:
Let's remove this property from this PR. This is going to be our next feature, and we will work with it in our next PR.

Reply:
Let's add it in next features then!

Reply:
removed in b764a48 !